packages/ssi-sdk-core/src/utils/image.ts

```diff
@@ -1,25 +1,67 @@
 import fetch from 'cross-fetch'
-import sizeOf from 'image-size'
-import { ISizeCalculationResult } from 'image-size/dist/types/interface'
+import { imageSize } from 'image-size'
 import { IImageDimensions, IImageResource } from '../types'
 
+type SizeCalculationResult = {
+  width?: number
+  height?: number
+  orientation?: number
+  type?: string
+}
+
+const uint8ArrayToString = (uint8Array: Uint8Array): string => {
+  const decoder = new TextDecoder('utf-8')
+  return decoder.decode(uint8Array)
+}
+
+// TODO: here we're handling svg separately, remove this section when image-size starts supporting it in version 2
+const isSvg = (uint8Array: Uint8Array): boolean => {
+  const text = uint8ArrayToString(uint8Array)
```

Comment:
Can't we detect this based in the array itself, or at least only take a few indices first. This probably has a massive hit on performance. This for instance will covert 10 MB images into text first, only to determine first whether it is a SVG or not

Reply:
yeah, changed the code to use only first bytes